configure.ac

```diff
@@ -451,10 +455,10 @@ Building Charliecloud
     C99 compiler ... ${CC} ${CC_VERSION}
 
   documentation: ${have_docs}
-    sphinx-build(1) $vmin_sphinx ... ${SPHINX:-not found} ${SPHINX_VERSION}
+    sphinx-build(1) ≥ $vmin_sphinx ... ${SPHINX_VERSION_NOTE}
     sphinx-build(1) Python ... ${sphinx_python:-n/a}
-    "docutils" module $vmin_docutils ... ${docutils_VERSION:-not found}
-    "sphinx-rtd-theme" module $vmin_rtd ... ${rtd_VERSION:-not found}
+    "docutils" module $vmin_docutils ≥ ... ${docutils_VERSION_NOTE}
+    "sphinx-rtd-theme" module $vmin_rtd ≥ ... ${rtd_VERSION_NOTE}
```

Comment:
"≥" is in the wrong place.

Reply:
Thanks! I had updated this while merging and misplaced it.